### PR TITLE
Add persist package and tool to the release note

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Comment by daneshk** [comment]
@sameerajayasoma @niveathika Please review

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -172,6 +172,13 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 
 - Added support for the new NATS JetStream client with publishing and subscribing functionalities
 
+##### `persist` package
+
+- Added a new `persist` package to provide the functionality required for Ballerina Persistence Layer.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:177]:
```suggestion
- Added a new `persist` package to provide the functionality required for the Ballerina persistence layer. 
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -172,6 +172,13 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 
 - Added support for the new NATS JetStream client with publishing and subscribing functionalities
 
+##### `persist` package
+
+- Added a new `persist` package to provide the functionality required for Ballerina Persistence Layer. 
+
+  Ballerina Persistent Layer provides functionality to store and query data conveniently.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:179]:
```suggestion
  The Ballerina persistent layer provides the functionality of storing and querying data conveniently.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -172,6 +172,13 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
...
+  This is an Experimental feature; APIs might change in future releases.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:180]:
```suggestion
  >**Info:** This is an experimental feature; APIs might change in future releases.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -215,6 +222,12 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 #### OpenAPI Tool
 Added support to generate Ballerina client and service declarations from Swagger 2.0(i.e. OpenAPI 2.0) definitions
 
+#### Persist Tool
+- Added a new `persist` command to make it easy to enable Ballerina Persistence Layer in a Ballerina project.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:226]:
```suggestion
- Added a new `persist` command to make it easy to enable the Ballerina persistence layer in a Ballerina project.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -215,6 +222,12 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
...
+
+  With this support, users can define an entity data model, validate the model and generate `persist` clients, which provide convenient APIs to store and query data in a data store.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:228]:
```suggestion
  With this support, users can define an entity data model, validate the model, and generate `persist` clients. This provides convenient APIs to store and query data in a data store.
```

**Review by praneesha — APPROVED** [review]

**Review by niveathika — APPROVED** [review]

**Review by sameerajayasoma — APPROVED** [review]

**Pull request merged**